modules/exploits/windows/local/bthpan.rb

```diff
@@ -125,7 +125,8 @@ def disclose_addresses(t)
   end
 
   def check
-    if sysinfo["Architecture"] =~ /wow64/i || sysinfo["Architecture"] =~ /x64/
+    # covers both native x64 and WOW64
+    if sysinfo["Architecture"] == ARCH_X64
```

Comment:
Definitely. `sysinfo["Architecture"]` no longer has the `WOW64` stuff in it, so it's always `x86` or `x64` now. This module isn't supported on 64-bit at all, so we only need to check the arch to know it's not supported.